core/block.js

```diff
@@ -1727,10 +1728,12 @@ Blockly.Block.prototype.removeInput = function(name, opt_quiet) {
       }
       input.dispose();
       this.inputList.splice(i, 1);
-      return;
+      return true;
     }
   }
-  if (!opt_quiet) {
+  if (opt_quiet) {
+    return false;
+  } else {
     throw Error('Input not found: ' + name);
```

Comment:
Even if it doesn't quite make sense, you'll have to return a value after the throw to satisfy the compiler.